metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -571,6 +695,21 @@ func createTimeSeries(resourceLabels map[string]string, metadata *metricMetadata
 	}
 }
 
+func createK8sTimeSeries(resource string, resourceLabels map[string]string, metadata *metricMetadata, point *sd_api.Point) *sd_api.TimeSeries {
```

Comment:
Same as above: make consistent with the metric names. New methods should be just methods and old methods should have a legacy prefix

Reply:
Done